tiled/server/authentication.py

```diff
@@ -894,11 +894,31 @@ async def refresh_session(
     return new_tokens
 
 
-@base_authentication_router.delete("/session/revoke/{session_id}")
+@base_authentication_router.post("/session/revoke")
 async def revoke_session(
+    request: Request,
+    refresh_token: schemas.RefreshToken,
+    settings: BaseSettings = Depends(get_settings),
+    db=Depends(get_database_session),
+):
+    "Mark a Session as revoked so it cannot be refreshed again."
+    request.state.endpoint = "auth"
+    payload = decode_token(refresh_token.refresh_token, settings.secret_keys)
+    session_id = payload["sid"]
+    # Find this session in the database.
+    session = await lookup_valid_session(db, session_id)
+    if session is None:
```

Comment:
This is a little picky, but...404 is probably not the right error code here. You're posting to something without specifying an ID...a client might interpret the 404 as saything that the path `/session/revoke` is not found. How about [409 Conflict](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409)?